src/Application.php

```diff
@@ -1324,8 +1324,12 @@ protected function getMethod()
     public function getPathInfo()
     {
         $query = isset($_SERVER['QUERY_STRING']) ? $_SERVER['QUERY_STRING'] : '';
-
-        return '/'.ltrim(str_replace('?'.$query, '', $_SERVER['REQUEST_URI']), '/');
+        $folder = dirname($_SERVER['SCRIPT_NAME']);
```

Comment:
Ya, if i run lumen in http://localhost/folder/folder/ then the $folder is **folder/folder/**
If run in root domain, the $folder is **/**